core/src/main/java/greencity/controller/UserController.java

```diff
@@ -560,6 +560,7 @@ public ResponseEntity<Long> findIdByEmail(@RequestParam String email) {
     public ResponseEntity<Object> updateUserLastActivityTime(@ApiIgnore @CurrentUser UserVO userVO,
         @PathVariable(value = "date") @DateTimeFormat(
             pattern = "yyyy-MM-dd.HH:mm:ss.SSSSSS") LocalDateTime userLastActivityTime) {
+        if(userVO.getRole() == Role.ROLE_USER) return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
```

Comment:
It is not required, You have already done it in SecurityConfig:

` .requestMatchers(HttpMethod.PUT,
                                "/ownSecurity/changePassword",
                                "/user/profile",
                                "/user/{id}/updateUserLastActivityTime/{date}",
                                "/user/language/{languageId}",
                                **"/user/updateUserLastActivityTime/{date}"**,
                                "/user/employee-email")
                        .hasAnyRole(USER, ADMIN, UBS_EMPLOYEE, MODERATOR, EMPLOYEE)`

Reply:
Perhaps, You have got the problem because of the security bug, which is fixed in Mariana`s pullRequest: [ [314] & [279] Fixed status response from 401 Unauthorized to 403 Forbidden in SecurityConfig, [PATCH /user/status], [GET /user/findUserForManagement]](https://github.com/GreenCityProject/GreenCityUser21_team2/pull/19)